lib/structures/Guild.js

```diff
@@ -172,7 +172,7 @@ class Guild extends Base {
                 if(!this.members.get(presence.user.id)) {
                     let userData = client.users.get(presence.user.id);
                     if(userData) {
-                        userData = `{username: ${userData.username}, id: ${userData.id}, discriminator: ${userData.discriminator}}`;
+                        userData = `{username: ${userData.username}, id: ${userData.id + (userData.discriminator === "0" ? "" : `, discriminator: ${userData.discriminator}`)}}`;
```

Comment:
No that's just a string for a debug log. This change just omits the users discrim from the log since it's not necessary anymore. Although this change isn't really necessary either, but if I didn't do this then it'd send `#0` inside the log

Reply:
Oh nevermind I see it now. It just looks a little messy imo. Would it be better if the whole discrim handling was done as its own separate thing either as an if or by doing `"id: ..." + discrim ternary` here?

Reply:
Or just let them see "0" anyways. It should be clear that the user was migrated to the new username system

Reply:
Reverted sir